`Error: Module did not self-register` when generating docs.

When working with docs, when I try to preview or even run the praxis:docs:install rake task, I get the following error.
```
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: Module did not self-register.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (~/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/bundler/gems/praxis-d8b0a68df075/lib/api_browser/node_modules/grunt-sass/node_modules/node-sass/lib/index.js:181:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
npm ERR! Linux 3.18.6-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v0.12.0
npm ERR! npm  v2.7.1
npm ERR! code ELIFECYCLE

npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.
```
As you can see there, I'm using node v0.12.0. Looking around it seems that one of the packages may not be compatible with this version of nodejs.
I see `"grunt-sass": "^0.17.0"` in the `lib/api_browser/package.json` file. These vesrions, with the issues I found ([gulp-sass issue](https://github.com/dlmanning/gulp-sass/issues/189) points to [node-sass issue](https://github.com/sass/node-sass/issues/653)), suggest that there may be some version dependency issues.

--- a/lib/api_browser/package.json
+++ b/lib/api_browser/package.json
@@ -23,7 +23,7 @@
     "grunt-file-blocks": "^0.3.0",
     "grunt-filerev": "^0.2.1",
     "grunt-ngmin": "0.0.3",
-    "grunt-sass": "^0.17.0",
+    "grunt-sass": "^0.18.0",
     "grunt-usemin": "^2.1.1",
     "grunt-wiredep": "^1.7.0",
     "load-grunt-tasks": "^0.4.0"

